.github/labeler.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2020 Dremio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Pull Request Labeler GitHub Action Configuration: https://github.com/marketplace/actions/labeler
+pr-native:
+  # Run native tests on any model or quarkus-related changes (may affect java reflection operation)
+  - model/**/*
```

Comment:
Why's model needed here?

Reply:
Mostly as a precaution. At some point we had to flag `NessieError` for reflection in native builds (#269)... It was a non-trivial dependency. I suppose it might be worth spending CI time on native build for model changes in general, because they are somewhat likely to be inspected via reflection.

Reply:
I'd be ok to just limit the model regex to `**/NessieError.java`